dags.yaml

```
@@ -1033,12 +1054,10 @@ bqetl_status_check:
   description: |
     This DAG checks if bigquery-etl is working properly. Dummy ETL tasks are executed to detect
     breakages as soon as possible.
-
     *Triage notes*
-
     None of these tasks should fail. If they do it is very likely that other/all ETL tasks will
     subsequently fail as well. Any failures should be communicated to the Data Infra Working Group
     as soon as possible.
   schedule_interval: "1h"
   tags:
-    - impact/tier_1
+    - impact/tier_1
```

Also, the newline at the end of the file got removed somehow, and the `build` CI check is failing because of it.
```suggestion
    - impact/tier_1

```